tensorboard/data/provider.py

```diff
@@ -421,7 +421,13 @@ class _TimeSeries(object):
     )
 
     def __init__(
-        self, max_step, max_wall_time, plugin_content, description, display_name
+        self,
```

Comment:
In the doc string of this class, please add that this class is also a super class of `TensorTimeSeries`.

Reply:
Thanks for the catch. That’s unrelated to this change, so I’ve done so
in #3290.